build.gradle

```diff
@@ -29,6 +30,7 @@ subprojects {
                 google_cloud_core            : "com.google.cloud:google-cloud-core:${googleCloudVersion}",
                 google_cloud_grpc            : "com.google.api.grpc:grpc-google-cloud-trace-v2:${googleCloudVersion}",
                 google_cloud_trace           : "com.google.cloud:google-cloud-trace:${googleCloudVersion}",
+                google_cloud_monitoring      : "com.google.cloud:google-cloud-monitoring:${cloudMonitoringVersion}",
```

Comment:
Yeah I used a separate version for cloud monitoring because the 1.0.2 version didn't have much and didn't work. I've added a separate `cloudTracingVersion` for tracing, kept it to 1.0.2 for this PR though.